src/simplewallet/simplewallet.cpp

```diff
@@ -5550,6 +5550,13 @@ bool simple_wallet::transfer_main(Transfer transfer_type, const std::vector<std:
   std::set<uint32_t> subaddr_indices  = {};
   if (!parse_subaddr_indices_and_priority(*m_wallet, local_args, subaddr_indices, priority)) return false;
 
+  if (priority == 0)
+  {
+    priority = m_wallet->get_default_priority();
+    if (priority == 0)
+      priority = transfer_type == Transfer::Locked ? tools::tx_priority_unimportant : tools::tx_priority_blink;
```

Comment:
This prevents users using blink to guarantee a locked transfer will get onto the blockchain, that feels slightly wrong to be removing that option.

Reply:
Just below this is a line that bails with an error if priority is set to blink for a locked transfer.

(I don't know why we impose that and agree it should go if there isn't a compelling reason, but needs to be separate from this PR).